src/common/settings.h

```diff
@@ -10,6 +10,7 @@
 #include <cstddef>
 #include <map>
 #include <string>
+#include <univalue.h>
```

Comment:
Every commit should pass compilation and all tests so if the failure occurs on the previous commit you should include it there (we are missing the "`CI / test each commit (pull_request)`" that exists on the other repos and I think @hebasto has it pending with other things he wanted to merge here into this repo).